panel-jinglitai-jlt4013a.c

```diff
@@ -96,6 +107,69 @@ static int jlt4013a_prepare(struct drm_panel *panel)
 	ST7701S_TEST(ret, st7701s_write_command(ctx, ST7701S_SLPOUT));
 	msleep(120);
 
+	/* BK0 */
+
+	ST7701S_TEST(ret, st7701s_write_command(ctx, ST7701S_CN2BKxSEL));
+	ST7701S_TEST(ret, st7701s_write_data(ctx, 0x77));
+	ST7701S_TEST(ret, st7701s_write_data(ctx, 0x01));
+	ST7701S_TEST(ret, st7701s_write_data(ctx, 0x00));
+	ST7701S_TEST(ret, st7701s_write_data(ctx, 0x00));
+	ST7701S_TEST(ret, st7701s_write_data(ctx, 0x10));
+
+	ST7701S_TEST(ret, st7701s_write_command(ctx, ST7701S_LNESET));	/* 854 as Table 12.3.2.7 */
+	ST7701S_TEST(ret, st7701s_write_data(ctx, 0xE9));
+	ST7701S_TEST(ret, st7701s_write_data(ctx, 0x03));
+
+	ST7701S_TEST(ret, st7701s_write_command(ctx, ST7701S_PORCTRL));
+	ST7701S_TEST(ret, st7701s_write_data(ctx, 0x11));
+	ST7701S_TEST(ret, st7701s_write_data(ctx, 0x02));
+
+	ST7701S_TEST(ret, st7701s_write_command(ctx, ST7701S_INVSET));
+	ST7701S_TEST(ret, st7701s_write_data(ctx, 0x31));
+	ST7701S_TEST(ret, st7701s_write_data(ctx, 0x03));
+
+	/* Not in documentation
+	ST7701S_TEST(ret, st7701s_write_command(ctx, 0xCC));
+	ST7701S_TEST(ret, st7701s_write_data(ctx, 0x10));
+	*/
+
+	ST7701S_TEST(ret, st7701s_write_command(ctx, ST7701S_PVGAMCTRL));
+	ST7701S_TEST(ret, st7701s_write_data(ctx, 0x40));
+	ST7701S_TEST(ret, st7701s_write_data(ctx, 0x01));
+	ST7701S_TEST(ret, st7701s_write_data(ctx, 0x46));
+	ST7701S_TEST(ret, st7701s_write_data(ctx, 0x0D));
+	ST7701S_TEST(ret, st7701s_write_data(ctx, 0x13));
+	ST7701S_TEST(ret, st7701s_write_data(ctx, 0x09));
+	ST7701S_TEST(ret, st7701s_write_data(ctx, 0x05));
+	ST7701S_TEST(ret, st7701s_write_data(ctx, 0x09));
+	ST7701S_TEST(ret, st7701s_write_data(ctx, 0x09));
+	ST7701S_TEST(ret, st7701s_write_data(ctx, 0x1B));
+	ST7701S_TEST(ret, st7701s_write_data(ctx, 0x07));
+	ST7701S_TEST(ret, st7701s_write_data(ctx, 0x15));
+	ST7701S_TEST(ret, st7701s_write_data(ctx, 0x12));
+	ST7701S_TEST(ret, st7701s_write_data(ctx, 0x4C));
+	ST7701S_TEST(ret, st7701s_write_data(ctx, 0x10));
+	ST7701S_TEST(ret, st7701s_write_data(ctx, 0xC8));
```

Comment:
I'm going to merge as-is but in the future I'll probably refactor this as a for-loop x2.